Gruntfile.js

```diff
@@ -220,7 +220,7 @@ module.exports = function(grunt) {
         exec: {
             revert: "./script/revert_pkg_version.pl",
             revert_release: "./script/revert_pkg_version.pl release",
-            deleteBuildFiles: "rm -r build",
+            deleteBuildFiles: "mkdir -p build && rm -r build",
```

Comment:
I guess this works, but it kind of seems perverse to create / delete, how about:

```bash
[ -d build ] && rm -r build
```

I think the intent is more  clear this way?